node/network/bridge/src/lib.rs

```diff
@@ -647,6 +854,39 @@ mod tests {
 		executor::block_on(future::select(test_fut, network_bridge));
 	}
 
+	async fn assert_sends_validation_event_to_all(
```

Comment:
```suggestion
	#[test]
	async fn assert_sends_validation_event_to_all(
```

Reply:
this isn't a test, but it's a test helper.